documentation/Linux.md

```diff
@@ -0,0 +1,63 @@
+# Installation Guide for Linux
+
+In order to use MIRAI, you need to install Rust, install Z3, and install MIRAI into cargo.
+
+## Installing Rust
+
+You should install rustup and then use it to get hold of the latest Rust compiler.
+See [here](https://doc.rust-lang.org/book/ch01-01-installation.html) for instructions.
+
+## Installing Z3
+
+On Fedora install it with
+```
+dnf install z3-devel
+```
+
+If that works, you're done. If not, you can find pre-built binaries for Z3 
+[here](https://github.com/Z3Prover/z3/releases). There are also binary libraries
+for linux included in the binaries directory of MIRAI.
+
+Alternatively you'll have to build the Z3 binaries yourself, 
+as described [here](https://github.com/facebookexperimental/MIRAI/blob/master/documentation/Z3AndLinux.md).
+
+## Installing MIRAI into cargo
+
+On Linux the z3 library has to be statically linked into the mirai executable. The best way to do that is
+clone the MIRAI repository:
+
+```
+cd ~
+git clone git@github.com:facebookexperimental/MIRAI.git
```

Comment:
I would not give them both options, just show the https one. Also you can probably omit the `cd ~`, delete the "The change to the MIRAI..." and just show it as
```
git clone https://...
cd MIRAI
```